tests/unit/models/reddit/test_collections.py

```diff
@@ -3,16 +3,15 @@
 
 from praw.models import Collection
 from praw.models.reddit.collections import SubredditCollections
-
 from ... import UnitTest
 
 
 class TestCollection(UnitTest):
     def test_eq(self):
         uuid = "fake_uuid"
         permalink = f"https://reddit.com/r/subreddit/collection/{uuid}"
-        collection1 = Collection(None, collection_id=uuid)
-        collection2 = Collection(None, permalink=permalink)
+        collection1 = Collection(self.reddit, collection_id=uuid)
```

Comment:
Why is `self.reddit` being passed now when it wasn't before? From what I understand, it isn't needed because unit tests don't make network requests.

Reply:
I changed these in order to match my IDE's type checker. It doesn't make a difference functionally, but by doing this then the Collection tests match the tests for other models.